backend/routes/google-oauth.ts

```diff
@@ -0,0 +1,17 @@
+import express from "express";
```

Comment:
nit: let's be consistent with file naming, since it looks like we use camel case throughout, let's rename this file to be `/routes/googleOAuth` 

that being said, this file in particular will likely contain future authentication related routes so it's up to you whether you want to keep this file solely for google oauth or rename to be more generic for all auth

Reply:
Changed file name to googleOAuth.

I think I will keep it only for google. I've thought it about it, when we implement other auth services, I think it will be easier to find with separate files.